components/gcp/bigquery/query/to_gcs/component.yaml

```diff
@@ -42,6 +42,10 @@ inputs:
     description: 'The path to the Cloud Storage bucket to store the query output.'
     default: ''
     type: GCSPath
+  - name: output_destination_format
```

Comment:
This input is not passed to the command-line program.

Reply:
Thanks for the review.
I forgot to add it.